index.html

```diff
@@ -195,35 +187,29 @@ <h2>Multi-Post
 		</section>
 		<section id="about_myself">
       <div class="myself">
-				<div>
-					<h1>About Myself</h1>
-				</div>
-				<div>
-					<p>
-						I’m a software developer! I can help you build a product
-						, feature or website Look through some of my work and
-						experience! If you like what you see and have a project
-						you need coded, don’t hesitate to contact me.
-					</p>
-				</div>
-				<div><p class="its_connect">LET’S CONNECT</p></div>
-				<div>
-					<ul class="social_icons">
-						<li><a href=""><img
-							class="icons"
-							src="./images/Twetter.svg"
-							alt="twetter"
-						/></a></li>
-						<li><a href=""><img
-							class="icons"
-							src="./images/Linkedin icon.svg"
-							alt="Linkedin"
-						/></a></li>
-						<li><a href=""><img class="icons" src="./images/madiam.svg" alt="madiam" /></a></li>
-						<li><a href=""><img class="icons" src="./images/github.svg" alt="github" /></a></li>
-						<li><a href=""><img class="icons" src="./images/victor.svg" alt="victor" /></a></li>
-					</ul>
-				</div>
+				<h1>About Myself</h1>
```

Comment:
In this section languages list need's to change display direction to horizontal on desktop kindly fix it 

![image (12)](https://github.com/sheezu900/My_Potfolio/assets/59707859/1793f212-01d7-4d06-b3d6-3534655c9832)
![image (13)](https://github.com/sheezu900/My_Potfolio/assets/59707859/7709bdff-ed15-4f92-83b3-ea93f91b4f4d)